### DISCUSSION TITLE
Supermaven not downloading binaries

### DISCUSSION THREAD

**Discussion by 90aca** [zed-industries/zed#15433] [Feature Requests]
Is there some special setup required so that Supermaven downloads the binary [it needs to run](https://github.com/zed-industries/zed/blob/main/crates/supermaven_api/src/supermaven_api.rs#L183)? It's currently just stuck at initializing, but nothing happens. There is also nothing in the logs. The .supermaven repository exists and the config.json is valid.

Version: Zed Preview 0.146.1
OS: MacOS Silicon

```json
  "features": {
    "inline_completion_provider": "supermaven"
  },
```

Some weeks ago it was working, but I manually deleted the .supermaven folder and now, even if the .supermaven folder exists, nothing is happening.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
